### ISSUE TITLE
Add functionality to override rules in VSCode

### ISSUE THREAD

**Issue by jhuiting** [Microsoft/vscode-tslint#240]
We've run into the issue that we want to disable certain rules in VSCode (like `
mocha-avoid-only`) that we still want to check on the CI. It would be nice to add a config setting that enables you to override certain rules from VSCode. The way we fixed it atm is by putting a duplicate config file in, which is kind of a waste.

If more people see value in this feature I'm attempting to add it to the vscode-tslint codebase :)

**Comment by carpben** [comment]
I agree. There are some rules, I want to be notified about in the command line, or I'd like tslint:fix to handle. Such as extra spaces at the end of line, or indentation issues. But I don't want to be notified about every extra space while I actually code. It would be nice if I could disable a tslint rule just for the editor.

**Comment by egamma** [comment]
@jhuiting having separate config files for different purposes is right way to do this. In particular since config files can extend each other and you do not need to duplicate settings. 

@carpben I can see that the squiggles get in the way when your violating a white space rule. One approach is to configure tslint to run on save only and not as you type.

Anyway since this implementation of tslint support is about to be deprecated, please file a feature request against the new implementation https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin.
